chromium-intent-to-prototype.md

```diff
@@ -0,0 +1,72 @@
+# Contact emails
```

Comment:
Can you rename the file (with .md or maybe chromium-i2p.md) and add the subject of the email to the content?